syntax = "proto3";

message Timeslot {
    string deptid = 1;
    string start = 2;
    string end = 3;
}

message GetTimeArg {
    string deptid = 1;
}

message TimeRsp{
    string yn = 1;
}

service TimeService {
    rpc CheckTime(GetTimeArg) returns (TimeRsp);
    rpc InsertTime(Timeslot) returns (Timeslot);
    rpc ClearTime(GetTimeArg) returns (TimeRsp);
}